MetasysServices/Audits/AuditInfoProvider.cs

```diff
@@ -34,39 +34,64 @@ public AuditInfoProvider(IFlurlClient client):base(client)
                                                "FlurlClient can not be null.");
         }
 
-        /// <inheritdoc/>
-        public async Task<PagedResult<AuditItemProvider>> GetAuditsAsync(AuditFilter auditFilter)
-        {                
-            return await GetPagedResultsAsync<AuditItemProvider>(BaseParam, ToDictionary(auditFilter));
-        }
-
-        /// <inheritdoc/>
+        /// <summary>
+        /// Retrieves the specified audit asynchronously.
+        /// </summary>
+        /// <param name="auditId">The identifier of the audit.</param>
+        /// <returns>The specified audit details.</returns>
         public async Task<AuditItemProvider> GetSingleAuditAsync(Guid auditId)
         {
-            var response=await GetRequestAsync("audits", null, auditId);
+            var response = await GetRequestAsync("audits", null, auditId);
             return JsonConvert.DeserializeObject<AuditItemProvider>(response.ToString());
-        }
+        }
 
-        /// <inheritdoc/>
-
+        /// <summary>
+        /// Retrieves a collection of audits asynchronously.
+        /// </summary>
+        /// <param name="auditFilter">The audit model to filter audits.</param>
+        /// <returns>The list of audits 
+        public async Task<PagedResult<AuditItemProvider>> GetAuditsAsync(AuditFilter auditFilter)

```

Comment:
Please remove comments for all methods and leave it with inheritdoc as it was before. This method is inheriting from IProvideAuditInfo and there is comments for all the method. see https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/inheritdoc